roachtest: psycopg failed

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1862644&tab=buildLog) on [release-19.1@c406bb10543ca97010c64cc230a3c45690a7eb6c](https://github.com/cockroachdb/cockroach/commits/c406bb10543ca97010c64cc230a3c45690a7eb6c):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200409-1862644/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.8-47-gc406bb1
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1862644&tab=artifacts#/psycopg)
Related:
- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1879134&tab=buildLog) on [release-19.1@86a7b6292f6d32dae066b46639ace2d30d8a2c96](https://github.com/cockroachdb/cockroach/commits/86a7b6292f6d32dae066b46639ace2d30d8a2c96):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.8-51-g86a7b62
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1879134&tab=artifacts#/psycopg)
Related:
- #47293 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1909769&tab=buildLog) on [release-19.1@eb23e9f18d425d8ebd3160c334e2649e954252f2](https://github.com/cockroachdb/cockroach/commits/eb23e9f18d425d8ebd3160c334e2649e954252f2):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.8-66-geb23e9f
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1909769&tab=artifacts#/psycopg)
Related:
- #47659 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004171855_v20.1.0-rc.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004171855_v20.1.0-rc.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47293 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1916036&tab=buildLog) on [release-19.1@d556976a57c52e188157469ec9a64d8f388a79e9](https://github.com/cockroachdb/cockroach/commits/d556976a57c52e188157469ec9a64d8f388a79e9):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.8-69-gd556976
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1916036&tab=artifacts#/psycopg)
Related:
- #47659 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004171855_v20.1.0-rc.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004171855_v20.1.0-rc.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47293 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1964572&tab=buildLog) on [release-19.1@b07422a7899db36972dbf36309677e7697f4acbb](https://github.com/cockroachdb/cockroach/commits/b07422a7899db36972dbf36309677e7697f4acbb):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.9-8-gb07422a
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1964572&tab=artifacts#/psycopg)
Related:
- #49265 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202005182011_v20.1.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202005182011_v20.1.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1967593&tab=buildLog) on [release-19.1@cd9ecd90d2ce0f5caf362d6ffa6f782e91640837](https://github.com/cockroachdb/cockroach/commits/cd9ecd90d2ce0f5caf362d6ffa6f782e91640837):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.9-11-gcd9ecd9
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1967593&tab=artifacts#/psycopg)
Related:
- #49265 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202005182011_v20.1.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202005182011_v20.1.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1980374&tab=buildLog) on [release-19.1@73a373fb8c138c8ef6e4a05d7c1757207efa0a8d](https://github.com/cockroachdb/cockroach/commits/73a373fb8c138c8ef6e4a05d7c1757207efa0a8d):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.9-15-g73a373f
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blacklist (psycopgBlackList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=1980374&tab=artifacts#/psycopg)
Related:
- #49265 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202005182011_v20.1.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202005182011_v20.1.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2030412&tab=buildLog) on [release-19.1@7101af9b438f1ae4a4b4557922ab73f95eef137e](https://github.com/cockroachdb/cockroach/commits/7101af9b438f1ae4a4b4557922ab73f95eef137e):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:753: 
		Tests run on Cockroach v19.1.9-21-g7101af9
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blocklist (psycopgBlockList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=2030412&tab=artifacts#/psycopg)
Related:
- #50490 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006220937_v19.2.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006220937_v19.2.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50140 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006091546_v20.1.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006091546_v20.1.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50139 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006032224_v20.2.0-alpha.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006032224_v20.2.0-alpha.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2054022&tab=buildLog) on [release-19.1@0c04a92ba19eedd4762ca7feb8361433682f3ded](https://github.com/cockroachdb/cockroach/commits/0c04a92ba19eedd4762ca7feb8361433682f3ded):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:757: 
		Tests run on Cockroach v19.1.10-5-g0c04a92
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blocklist (psycopgBlockList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=2054022&tab=artifacts#/psycopg)
Related:
- #50799 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2060752&tab=buildLog) on [release-19.1@0c04a92ba19eedd4762ca7feb8361433682f3ded](https://github.com/cockroachdb/cockroach/commits/0c04a92ba19eedd4762ca7feb8361433682f3ded):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:757: 
		Tests run on Cockroach v19.1.10-5-g0c04a92
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blocklist (psycopgBlockList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=2060752&tab=artifacts#/psycopg)
Related:
- #50799 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2068280&tab=buildLog) on [release-19.1@2838bf599a5abec871b8812ef66d1dd5ed75a4d7](https://github.com/cockroachdb/cockroach/commits/2838bf599a5abec871b8812ef66d1dd5ed75a4d7):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:757: 
		Tests run on Cockroach v19.1.10-7-g2838bf5
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blocklist (psycopgBlockList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=2068280&tab=artifacts#/psycopg)
Related:
- #51017 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292108_v19.1.11](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292108_v19.1.11) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50799 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).psycopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2072432&tab=buildLog) on [release-19.1@8ecf958ac06ee10391ceb108ba11a745de8ff4b1](https://github.com/cockroachdb/cockroach/commits/8ecf958ac06ee10391ceb108ba11a745de8ff4b1):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/psycopg/run_1
	orm_helpers.go:214,orm_helpers.go:144,psycopg.go:121,psycopg.go:134,test_runner.go:757: 
		Tests run on Cockroach v19.1.11-6-g8ecf958
		Tests run against psycopg 2_8_5
		676 Total Tests Run
		384 tests passed
		292 tests failed
		95 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorRealTests.test_copy (unexpected)
		--- FAIL: tests.test_cursor.CursorTests.test_description_slice (unexpected)
		--- FAIL: tests.test_extras_dictcursor.ExtrasDictCursorTests.test_copy (unexpected)
		For a full summary look at the psycopg artifacts 
		
		An updated blocklist (psycopgBlockList19_1) is available in the artifacts' psycopg log

```

<details><summary>More</summary><p>


Artifacts: [/psycopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=2072432&tab=artifacts#/psycopg)
Related:
- #51017 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292108_v19.1.11](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292108_v19.1.11) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50799 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #47262 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46900 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46822 roachtest: psycopg failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apsycopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/psycopg.go
+++ b/pkg/cmd/roachtest/psycopg.go
@@ -12,13 +12,16 @@ package main
 
 import (
 	"context"
+	"fmt"
 	"regexp"
 )
 
 var psycopgReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)(?:_(?P<minor>\d+)(?:_(?P<point>\d+)(?:_(?P<subpoint>\d+))?)?)?$`)
 
-// This test runs psycopg full test suite against a single cockroach node.
+// TODO(rafi): use a release tag once the commit below appears in a release.
+var supportedPsycopgTag = "cecff195fc17a83d593dd62c239aa188883a844e"
 
+// This test runs psycopg full test suite against a single cockroach node.
 func registerPsycopg(r *testRegistry) {
 	runPsycopg := func(
 		ctx context.Context,
@@ -48,6 +51,7 @@ func registerPsycopg(r *testRegistry) {
 			t.Fatal(err)
 		}
 		c.l.Printf("Latest Psycopg release is %s.", latestTag)
+		c.l.Printf("Supported Psycopg release is %s.", supportedPsycopgTag)
 
 		if err := repeatRunE(
 			ctx, c, node, "update apt-get", `sudo apt-get -qq update`,
@@ -77,8 +81,20 @@ func registerPsycopg(r *testRegistry) {
 			c,
 			"https://github.com/psycopg/psycopg2.git",
 			"/mnt/data1/psycopg",
-			latestTag,
+			"master",
 			node,
+		); err != nil {
+			t.Fatal(err)
+		}
+
+		// TODO(rafi): once there's a real release, change the clone step above, and
+		// remove this.
+		if err := repeatRunE(
+			ctx,
+			c,
+			node,
+			"checkout supported tag",
+			fmt.Sprintf(`cd /mnt/data1/psycopg/ && git checkout %s`, supportedPsycopgTag),
 		); err != nil {
 			t.Fatal(err)
 		}
@@ -120,7 +136,7 @@ func registerPsycopg(r *testRegistry) {
 		results.parsePythonUnitTestOutput(rawResults, expectedFailures, ignoredlist)
 		results.summarizeAll(
 			t, "psycopg" /* ormName */, blocklistName, expectedFailures,
-			version, latestTag,
+			version, supportedPsycopgTag,
 		)
 	}
 

